### PR TITLE
530 - Journey test for serving stipulated decision

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: web-client/integration-tests/signAndServeStipulatedDecision.test.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+  let docketNumber = null;
+  let signedDocumentId = null;
+  let caseDetail;
+
+  it('login as a tax payer and create a case', async () => {
+    await loginAs(test, 'taxpayer');
+    caseDetail = await uploadPetition(test);
+    // eslint-disable-next-line prefer-destructuring
+    docketNumber = caseDetail.docketNumber;
```
Comment (rachaelparris) [inline web-client/integration-tests/signAndServeStipulatedDecision.test.js]: `({ docketNumber } = caseDetail);` should resolve the destructuring eslint issue.

File: web-client/integration-tests/signAndServeStipulatedDecision.test.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  loginAs,
+  serveDocument,
+  setupTest,
+  signProposedStipulatedDecision,
+  uploadExternalDecisionDocument,
+  uploadPetition,
+  uploadProposedStipulatedDecision,
+  viewCaseDetail,
+  viewDocumentDetailMessage,
+} from './helpers';
```
Comment (rachaelparris) [inline web-client/integration-tests/signAndServeStipulatedDecision.test.js:13]: It looks like several of these imports aren't used, could you remove them?

File: web-client/integration-tests/signAndServeStipulatedDecision.test.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    await serveDocument({
+      docketNumber: signedStipulatedDecision.docketNumber,
+      documentId: signedStipulatedDecision.document.documentId,
+      messageId: signedStipulatedDecision.currentMessage.messageId,
+      test,
+      workItemIdToMarkAsRead: signedStipulatedDecision.workItemId,
+    });
+  });
+
+  it('the case status should become closed', async () => {
```
Comment (rachaelparris) [inline web-client/integration-tests/signAndServeStipulatedDecision.test.js:119]: Is the case status actually updated to `Closed`? If so, can you test for that here?

**Review by rachaelparris — APPROVED** [review]
Found a few minor things

**Review by sutt0n — APPROVED** [review]

**Pull request merged**